tools/codesigningtool/codesigningtool.py

```diff
@@ -0,0 +1,84 @@
+import argparse
```

Comment:
please add the copyright notice as seen in other python tools, like: https://github.com/bazelbuild/rules_apple/blob/59050b3aa918dc4ce6b6f4d54da58a35f2486bb2/tools/bundletool/bundletool.py

Reply:
👍